google/resource_google_organization_policy.go

```diff
@@ -227,6 +248,20 @@ func flattenBooleanOrganizationPolicy(policy *cloudresourcemanager.BooleanPolicy
 	return bPolicies
 }
 
+func flattenRestoreOrganizationPolicy(restore_policy *cloudresourcemanager.RestoreDefault) []map[string]interface{} {
+	bRestore_policy := make([]map[string]interface{}, 0, 1)
```

Comment:
I'm not sure what the `b` prefix means here. Maybe just call it `rp` or something simple like that?